frontend/doc-site/032__Contribute and Publish Data.mdx

```diff
@@ -22,14 +22,14 @@ CELLxGENE supports most single-cell RNA-seq and ATAC-seq data, but a few types o
 - species not on the [supported list](https://chanzuckerberg.github.io/single-cell-curation/latest-schema.html#organism_ontology_term_id)
 - assays not on the [supported list](https://github.com/chanzuckerberg/cellxgene-census/blob/main/docs/census_accepted_assays.csv)
   - these additional assays are pending Census acceptance and will be accepted:
```

Comment:
This is slightly confusing. What does "and will be accepted" mean?

Reply:
Opted for a more concise "these additional assays are accepted"